src/liballoc/borrow.rs

```diff
@@ -141,6 +141,40 @@ impl<T> ToOwned for T
 /// let mut input = Cow::from(vec![-1, 0, 1]);
 /// abs_all(&mut input);
 /// ```
+///
```

Comment:
Please add an "introduction" sentence explaining what does this new example (otherwise it's strange to just have two examples following each other).

Reply:
Done